architecture/core_concepts/image_streams.adoc

```diff
@@ -0,0 +1,614 @@
+[[architecture-core-concepts-image-streams]]
+= Image Streams
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+ifdef::atomic-registry[]
+== Image Streams
+endif::[]
+
+
+[[image-streams]]
+//from https://blog.openshift.com/image-streams-faq/
+An _image stream is a pointer to an image using various identifying qualities. This means that even if the source image changes, 
+the image stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+Note that an image stream can be configured to periodically chack for updates (using the `--scheduled` flag on a tag) 
+or you can re-importing an image (using the `oc import-image` command). 
+
+An image stream does not contain actual image data, it is simply a mapping between tags and actual images. 
+It can contain additional metadata that describes the tags provided by the image stream.
+
+The images can be stored in any of the following:
+
+. {product-title}'s xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
+registry]
+. An external registry, such as `registry.access.redhat.com`, `hub.docker.com`, and so forth.
+. Other image streams
+
+An _image stream_ comprises any number of
+xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
+container images] identified by tags. It presents a single virtual view of
+related images, similar to an image repository. 
+
+== Important terms 
+
+Docker repository::
+A central location for uploading and downloading images.
+For example, the OpenShift Jenkins images are in a Docker registry: *docker.io/openshift/jenkins-2-centos7:latest*.
```

Comment:
since this glossary appears twice i'm not sure what i'm reviewing anymore. i'm stopping here.

Reply:
@bparees Sorry. This comment must have gotten lost. I was thinking of breaking the Builds and Image Stream topic into two topics. This file is a test and I have not updated it for some time.

Reply:
can you create a PR where I can actually see the proposed changes, then?  this is really frustrating to try to review in its current state.

Reply:
@bparees All proposed changes are in architecture/core_concepts/builds_and_image_streams.adoc. I removed the extraneous document.